tests/test_stream_engine.py

```diff
@@ -95,6 +96,8 @@ async def stream(_):
             Consumer.start.assert_awaited()
             stream_engine._producer.start.assert_awaited()
 
+            await asyncio.sleep(0)  # Allow stream coroutine to run once
```

Comment:
Got warnings about the stream coroutine never being awaited, this was because the task would be cancelled before it got the time to run on the event loop.
This sleep passes context to the event loop to let the task with the stream coroutine in it run